### DISCUSSION TITLE
Problem with scss modules

### DISCUSSION THREAD

**Discussion by fraitag** [RyanClementsHax/storybook-addon-next#128] [Q&A]
Hi i have problem with scss module when I using object desctructing on import.

For instance
`import styles, { image, backgroundImage } from './Image.module.scss';`

`styles` return proper object with image and backgroundImage

but typeof `image`, `backgroundImage` is undefined

Do you have any ideas? In my project I use very offen object desctructing on import.

**Comment by RyanClementsHax** [comment]
Hiya! Thanks for opening this issue. Could you provide a reproduction repo so I could look into this further?
